src/utils/auth-helper.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Fetches config from environment and API.
+ * @returns A promise to get & set session storage keys with appropriate values.
+ */
+/** Gets Keycloak JWT and parses it. */
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+
+export function getJWT (): any {
```

Comment:
This function doesn't need to be exported.

Reply:
Updated in next commit 👍